test/e2e/common/runtime.go

```diff
@@ -41,7 +41,7 @@ var _ = framework.KubeDescribe("Container Runtime", func() {
 			/*
 				Release : v1.13
 				Testname: Container Runtime, Restart Policy, Pod Phases
-				Description: If the restart policy is set to ‘Always’, Pod MUST be restarted when terminated, If restart policy is ‘OnFailure’, Pod MUST be started only if it is terminated with non-zero exit code. If the restart policy is ‘Never’, Pod MUST never be restarted. All these three test cases MUST verify the restart counts accordingly.
+				Description: If the restart policy is set to 'Always', Pod MUST be restarted when terminated, If restart policy is 'OnFailure', Pod MUST be started only if it is terminated with non-zero exit code. If the restart policy is 'Never', Pod MUST never be restarted. All these three test cases MUST verify the restart counts accordingly.
```

Comment:
this requires changing the conformance.txt file as well... see the verify CI job failure

Reply:
thank you for your comment. sorry, I can't find error messages about conformance.txt in https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/87680/pull-kubernetes-verify/1222686544863694851 could you please show me the point?

Reply:
sorry, I was wrong... the description isn't pulled into test/conformance/testdata/conformance.txt